src/applications/ivc-champva/10-7959C/helpers/utilities.js

```diff
@@ -9,6 +9,8 @@ export function isRequiredFile(formContext, requiredFiles) {
 // Return either 'your' or the applicant's name depending
 export function nameWording(formData, isPosessive = true, cap = true) {
   let retVal = '';
+  // NOTE: certifierRole isn't used in this form anymore so this will always
+  // skip to else clause
   if (formData?.certifierRole === 'applicant') {
     retVal = isPosessive ? 'your' : 'you';
   } else {
```

Comment:
Despite not being used, leaving this here because we may bring back a similar screener value (`certifierName`) in the future if we decide we need to track a separate name for the person filling out the form.